README.md

```diff
@@ -11,3 +11,18 @@ _**Prerequisits: This repository requires node v16 and up**_
 1. Clone this repository
 2. Install dependecies with `npm i`
 3. Test and lint with `npm test`
+
+## Manual Deployment
```

Comment:
This deploys the current commit to a uniquly generated URL, right? Maybe this is a more accurate title:

```suggestion
## Deploy Local Commits
```